tests/unit/test_simpleworkflow.py

```diff
@@ -280,5 +280,18 @@ def test_multiplex_output_as_another_input(self):
         dstprefixoutput = [output for output in outputs if output['name'] == 'DST_multiplex_prefix']
         assert len(dstprefixoutput) == 1
 
+    @vcr.use_cassette('tests/unit/cassettes/test_batch_workflows_works.yaml', record_mode='new_episodes', filter_headers=['authorization'])
+    def test_batch_workflows_works(self):
+        """
+        submit a workflow with multiple params, hit batch workflow endpoint
+        :return:
+        """
+        # note there are 2 inputs
+        data = ["s3://receiving-dgcs-tdgplatform-com/054813633050_01_003",
+                "http://test-tdgplatform-com/data/QB02/LV1B/053702625010_01_004/053702625010_01/053702625010_01_P013_MUL"]
+        aoptask = self.gbdx.Task("AOP_Strip_Processor", data=data, enable_acomp=True, enable_pansharpen=True)
+        workflow = self.gbdx.Workflow([aoptask])
+        workflow.savedata(aoptask.outputs.data, location='some_folder')
+        batch_workflow_id = workflow.execute()
```

Comment:
please
* verify the batch_workflow_id is a relatively valid id  ( has sufficient length or something )
* that workflow.status works and verify its contents
* that the above-mentioned statuses work (workflow.running, workflow.succeeded, workflow.complete)